ci/axis/dask.yaml

```diff
@@ -41,7 +41,7 @@ excludes:
   - DOCKER_FILE: dask_sql.Dockerfile
     BUILD_IMAGE: gpuci/distributed
 
-  - RAPIDS_VER: '21.08'
-    UCX_PY_VER: '0.22'
   - RAPIDS_VER: '21.10'
-    UCX_PY_VER: '0.21'
\ No newline at end of file
+    UCX_PY_VER: '0.23'
+  - RAPIDS_VER: '21.12'
+    UCX_PY_VER: '0.22'
```

Comment:
`UCX_PY_VER` is swapped, 21.10->0.22, 21.12->0.23.

Reply:
This section is excluding these builds from the matrix - essentially, it's saying "don't build RAPIDS 21.10 with ucx-py 0.23, and don't build 21.12 with 0.22."

IMO, this section is confusing and creates a lot of toil, which was the rationale behind #7, which attempts to compute all of this in the actual build file so that updating this file is more intuitive.

Reply:
Got it, indeed it's confusing, but I agree it's correct then. Sorry for the noise.